ql/src/java/org/apache/hadoop/hive/ql/exec/DataSketchesFunctions.java

```diff
@@ -0,0 +1,218 @@
+package org.apache.hadoop.hive.ql.exec;
+
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDAFResolver2;
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDTF;
+
+public class DataSketchesFunctions {
+
+  private static final String DATA_TO_SKETCH = "datatosketch";
+  private static final String SKETCH_TO_ESTIMATE_WITH_ERROR_BOUNDS = "sketchToEstimateWithErrorBounds";
+  // FIXME: consider to rename it to simply "estimate" or "evaluate" - in case of the counting sketches the "sketchto..." doesnt add value
+  private static final String SKETCH_TO_ESTIMATE = "sketchToEstimate";
+  private static final String SKETCH_TO_STRING = "sketchToString";
+  // FIXME: probably use simply "union" instead unionSketch?
+  private static final String UNION_SKETCH = "unionSketch";
+  private static final String GET_N = "getN";
+  private static final String GET_CDF = "getCdf";
+  private static final String GET_PMF = "getPmf";
+  private static final String GET_QUANTILES = "GetQuantiles";
+  private static final String GET_QUANTILE = "GetQuantile";
+  private static final String GET_RANK = "getRank";
+  private static final String INTERSECT_SKETCH = "intersection";
+  private static final String EXCLUDE_SKETCH = "exclude";
+  private static final String GET_K = "getK";
+  private static final String GET_FREQUENT_ITEMS = "getFrequentItems";
+  private static final String T_TEST = "TTest";
+  private static final String SKETCH_TO_MEANS = "sketchtomeans";
```

Comment:
As you mentioned above, probably `sketchto` can be removed from these function names. Additionally, we can use `_` when they consist of multiple words.